lib/facilities/cemetery_data/cems.xml

```diff
@@ -302,7 +303,7 @@ contact2=""
 phone="707-944-4600"
 fax=""
 email=""
-lat="38.390938"
+lat=" 38.390938"

```

Comment:
leading space.. not sure if that's a problem or not

Reply:
Whoops. Fixing.